inc/config_cascade.php

```diff
@@ -61,6 +61,10 @@
         'userscript' => array(
             'default'   => array(DOKU_CONF . 'userscript.js')
         ),
+        'styleini' => array(
+            'default'   => array(DOKU_INC . 'lib/tpl/TPL_PLACEHOLDER/' . 'style.ini'),
+            'local'     => array(DOKU_CONF . 'tpl/TPL_PLACEHOLDER/' . 'style.ini')
```

Comment:
We will want a more obvious placeholder name here. How about `%TEMPLATE%`?

Reply:
I am using %TEMPLATE% in the latest commit.